src/ServiceManagement/Compute/Commands.ServiceManagement/IaaS/Network/NewAzureReservedIP.cs

```diff
@@ -26,9 +26,17 @@ namespace Microsoft.WindowsAzure.Commands.ServiceManagement.IaaS
     [Cmdlet(VerbsCommon.New, ReservedIPConstants.CmdletNoun, DefaultParameterSetName = ReserveNewIPParamSet), OutputType(typeof(ManagementOperationContext))]
     public class NewAzureReservedIPCmdlet : ServiceManagementBaseCmdlet
     {
-        protected const string ReserveNewIPParamSet = "CreateNewReservedIP";
-        protected const string ReserveInUseIPUsingSlotParamSet = "CreateInUseReservedIPUsingSlot";
-        protected const string ReserveInUseIPParamSet = "CreateInUseReservedIP";
+        public const string ReserveNewIPParamSet = "CreateNewReservedIP";
+        public const string ReserveInUseIPUsingSlotParamSet = "CreateInUseReservedIPUsingSlot";
+        public const string ReserveInUseIPParamSet = "CreateInUseReservedIP";
+
+        public NewAzureReservedIPCmdlet()
```

Comment:
Added a parameterized constructor, so default constructor is required to support backward compatibility.